cleanco/__init__.py

```diff
@@ -1 +1,3 @@
 from .cleanco import cleanco
+from .clean import prepare_terms, basename
+from .classify import typesources, countrysources, matches
```

Comment:
If anything, what should be importable at top-level for type & country classification now?

Reply:
classifierList() from classify.py should tell you all of the possible classifiers from the iso20275 library through a list. Some have more data than others per the spec, but 'country' and 'local_name' are the ones that match from before.

idClassification() replaces both typesources() and countrysources() in classify.py.

Reply:
Did you want to try to retain the backwards compatibility here? It is just a matter of converting those strings, and I can take a look.

Reply:
The package has always supported checking for jurisdiction and legal form. I guess it would be good to maintain that capability?

Reply:
Backwards compatibility retained.